content/development/regex_assembly.md

```diff
@@ -356,22 +357,25 @@ it{{quotes}}s{{opt-lazy-wspace}}possible
 
 Note that the include processor does not have a body, thus the end marker is optional.
 
+Please see [Include-Except processor]({{< ref "regex_assembly#include-except-processor" >}}) for how suffix replacements work.
+
 ## Include-Except processor
 
 Processor name: `include-except`
 
 ### Arguments
 
 - Include file name (required): The name of the file to include, without suffix
-- Exclude file name (required): The name of the file to consult for exclusions, without suffix
+- Exclude file names (required): One or more names of files to consult for exclusions, without suffix, space separated
+- Suffix replacements (optional): Any number of space separated two-tuples, where the first entry is the suffix to match and the second entry is the replacement. To use, end the list of exclude file names with `--`. Tuples are space separated
```

Comment:
Same comment as above regarding repetition: might be an oversight or maybe it's intentional.